workflow/hosts/jet.yaml

```diff
@@ -0,0 +1,23 @@
+BASE_GIT: '/lfs4/HFIP/hfv3gfs/glopara/git'
+DMPDIR: '/lfs4/HFIP/hfv3gfs/glopara/dump'
+PACKAGEROOT: '/lfs4/HFIP/hfv3gfs/glopara/nwpara'
+COMROOT: '/lfs4/HFIP/hfv3gfs/glopara/com'
+COMINsyn: '${COMROOT}/gfs/prod/syndat'
+HOMEDIR: '/lfs4/HFIP/hfv3gfs/${USER}'
+STMP: '/lfs4/HFIP/hfv3gfs/${USER}/stmp'
+PTMP: '/lfs4/HFIP/hfv3gfs/${USER}/ptmp'
+NOSCRUB: $HOMEDIR
+ACCOUNT: hfv3gfs
+SCHEDULER: slurm
+QUEUE: batch
+QUEUE_SERVICE: batch
+PARTITION_BATCH: xjet
+PARTITION_SERVICE: service
+CHGRP_RSTPROD: 'YES'
+CHGRP_CMD: 'chgrp rstprod'
+HPSSARCH: 'YES'
+LOCALARCH: 'NO'
+ATARDIR: '/NCEPDEV/${HPSS_PROJECT}/1year/${USER}/${machine}/scratch/${PSLOT}'
+MAKE_NSSTBUFR: 'NO'
+MAKE_ACFTBUFR: 'NO'
+SUPPORTED_RESOLUTIONS: ['C96', 'C48']
```

Comment:
Are the `SUPPORTED_RESOLUTIONS` values up-to-date?

Reply:
Yes, currently these are all that I have tested. I will update this as I test higher resolutions.

Reply:
Gotcha, sounds good!